pkgs/top-level/all-packages.nix

```diff
@@ -7823,6 +7823,8 @@ in
 
   inherit (callPackages ../development/compilers/graalvm { }) mx jvmci8 graalvm8;
 
+  graalvm8-ee = callPackages ../development/compilers/graalvm/enterprise-edition.nix { };
```

Comment:
`callPackages` is wrong here. This PR causes warnings during Nixpkgs evaluation.

Reply:
Fixed in  7d9d632b5ed60. It's very hard to catch this since it's a difference of one letter.
I think we need a check for it, I've made this mistake numerous occasions.

Reply:
>  it's a difference of one letter.